pkg/kubelet/kubelet.go

```diff
@@ -78,19 +78,23 @@ const (
 )
 
 var (
-	// ErrNoKubeletContainers returned when there are not containers managed by the kubelet (ie: either no containers on the node, or none that the kubelet cares about).
+	// ErrNoKubeletContainers returned when there are not containers managed by
+	// the kubelet (ie: either no containers on the node, or none that the kubelet cares about).
 	ErrNoKubeletContainers = errors.New("no containers managed by kubelet")
 
-	// ErrContainerNotFound returned when a container in the given pod with the given container name was not found, amongst those managed by the kubelet.
+	// ErrContainerNotFound returned when a container in the given pod with the
+	// given container name was not found, amongst those managed by the kubelet.
 	ErrContainerNotFound = errors.New("no matching container")
 )
 
 // SyncHandler is an interface implemented by Kubelet, for testability
 type SyncHandler interface {
+
 	// Syncs current state to match the specified pods. SyncPodType specified what
 	// type of sync is occuring per pod. StartTime specifies the time at which
 	// syncing began (for use in monitoring).
-	SyncPods(pods []api.BoundPod, podSyncTypes map[types.UID]metrics.SyncPodType, startTime time.Time) error
+	SyncPods(pods []api.BoundPod, podSyncTypes map[types.UID]metrics.SyncPodType, mirrorPods util.StringSet,
```

Comment:
Can we document what mirrorPods contains and whether they are filtered from pods?